lib/sshutils/scp/http.go

```diff
@@ -269,3 +274,17 @@ type nopWriteCloser struct {
 func (wr *nopWriteCloser) Close() error {
 	return nil
 }
+
+const (
+	// FileTransferRequest is an optional parameter id of an file transfer request that has gone through
+	// an approval process during a moderated session to allow a file transfer scp command to be executed
+	// used as a value in the file transfer context and env var for exec session
+	FileTransferRequest ContextKey = "FILE_TRANSFER_REQUEST_ID"
```

Comment:
I think better name would be `FileTransferRequestID` to more accurately show what it is and be consistent with the rest of the naming. (Same for `ModeratedSession`)